src/app/Mappings/components/MappingBuilder/helpers.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  MappingSource,
+  Mapping,
+  MappingItem,
+  MappingType,
+  MappingTarget,
+  INetworkMappingItem,
+  IStorageMappingItem,
+} from '../../types';
+import { IMappingBuilderItem } from './MappingBuilder';
+import { IVMwareProvider, ICNVProvider } from '@app/Providers/types';
+
+export const getSourceById = (sources: MappingSource[], id: string): MappingSource | undefined =>
+  sources.find((source) => source.id === id);
+
+export const getBuilderItemsFromMapping = (
```

Comment:
Is this ever called?

Reply:
Not yet, but it can be used to pre-populate the state here from API mappings when we add edit functionality.